runtime/vm/FastJNI_java_lang_Throwable.cpp

```diff
@@ -58,13 +58,12 @@ Fast_java_lang_Throwable_fillInStackTrace(J9VMThread *currentThread, j9object_t
 				walkState->restartException = receiver;
 			}
 			walkState->flags = walkFlags;
+			walkState->skipCount = 1;	/* skip the INL frame */
 #if JAVA_SPEC_VERSION >= 15
 			J9Class *receiverClass = J9OBJECT_CLAZZ(currentThread, receiver);
```

Comment:
@gacholio did you mean this variable declaration? it is within a `.cpp` file and not required to be declared at beginning of the scope.

Reply:
You are correct - the C change still has the scope, which I missed.